config/lava/chromeos/tast.jinja2

```diff
@@ -0,0 +1,30 @@
+- test:
+      timeout:
+        minutes: 45
+      docker:
+        image: {{ docker_image }}
+        wait:
+          device: true
+      results:
+          location: /home/cros-tast/lava
+      definitions:
+      - repository:
+          metadata:
+            format: Lava-Test Test Definition 1.0
+            name: docker-ping
```

Comment:
Wrong name

Reply:
Fixed and tested in https://lava.collabora.co.uk/scheduler/job/5987415